Octokit.Tests.Integration/Clients/RepositoriesClientTests.cs

```diff
@@ -534,6 +534,19 @@ public class TheGetMethod
         }
     }
 
+    public class TheGetAllPublicMethod
+    {
+        [IntegrationTest]
```

Comment:
Of course. I will do that.

On Tue, Feb 24, 2015 at 9:07 AM, Brendan Forster <notifications@github.com>
wrote:

> In Octokit.Tests.Integration/Clients/RepositoriesClientTests.cs
> <https://github.com/octokit/octokit.net/pull/691#discussion_r25237269>:
>
> > @@ -534,6 +534,19 @@ public class TheGetMethod
> >          }
> >      }
> >
> > +    public class TheGetAllPublicMethod
> > +    {
> > +        [IntegrationTest]
>
> If you could throw a Skip on this test, it'd be greatly appreciated -
> this test is going to run for a *long, long, long* time
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/octokit/octokit.net/pull/691/files#r25237269>.
>